### PR TITLE
Remove personal email address and other minor nits

### PR DESCRIPTION
**Pull request by niks3089**
This change is [Reviewable](https://reviewable.io/reviews/nimbella/command-sets/58)

### CODE REVIEW

File: bluejeans/README.md [hunk 1]
```diff
@@ -16,10 +16,11 @@ After you've the credentials, we need two secrets named `bluejeansAppKey` & `blu
 
 ## Usage
 
-To create a meeting with `satya@nimbella.com` & `eric@nimbella.com`:
+To create a meeting with `joe@commander.com` & `judy@commander.com`:
```
Comment (rabbah) [inline bluejeans/README.md]:
```suggestion
To create a meeting with `joe@example.com` & `judy@example.com`:
```

File: bluejeans/README.md [hunk 2]
```diff
@@ -16,10 +16,11 @@ After you've the credentials, we need two secrets named `bluejeansAppKey` & `blu
...
 ```sh
-/dapp bluejeans_create -title "A new beginning" -emails "satya@nimbella.com,eric@nimbella.com" -start "03/01/20 18:00" -end "03/01/20 18:30'
+/dapp bluejeans_create -title "A new beginning" -emails "joe@commander.com,judy@commander.com" -start "03/01/20 18:00" -end "03/01/20 18:30'
```
Comment (rabbah) [inline bluejeans/README.md]:
```suggestion
/dapp bluejeans_create -title "A new beginning" -emails "joe@example.com,judy@example.com" -start "03/01/20 18:00" -end "03/01/20 18:30'
```

File: bluejeans/src/bluejeans.js [hunk 3]
```diff
@@ -24,10 +24,11 @@ After you've the credentials, we need two secrets named \`bluejeansAppKey\` & \`
 
 ## Usage
 
-To create a meeting with \`satya@nimbella.com\` & \`eric@nimbella.com\`:
+To create a meeting with \`joe@commander.com\` & \`judy@commander.com\`:
```
Comment (rabbah) [inline bluejeans/src/bluejeans.js]:
```suggestion
To create a meeting with \`joe@example.com\` & \`judy@example.com\`:
```

File: bluejeans/src/bluejeans.js [hunk 4]
```diff
@@ -24,10 +24,11 @@ After you've the credentials, we need two secrets named \`bluejeansAppKey\` & \`
...
 \`\`\`sh
-/dapp bluejeans_create -title "A new beginning" -emails "satya@nimbella.com,eric@nimbella.com" -start "03/01/20 18:00" -end "03/01/20 18:30'
+/dapp bluejeans_create -title "A new beginning" -emails "joe@commander.com,judy@commander.com" -start "03/01/20 18:00" -end "03/01/20 18:30'
```
Comment (rabbah) [inline bluejeans/src/bluejeans.js]:
```suggestion
/dapp bluejeans_create -title "A new beginning" -emails "joe@example.com,judy@example.com" -start "03/01/20 18:00" -end "03/01/20 18:30'
```

File: bluejeans/README.md [hunk 5]
```diff
@@ -16,10 +16,11 @@ After you've the credentials, we need two secrets named `bluejeansAppKey` & `blu
```
Comment (satyarohith) [inline bluejeans/README.md:16]: > `bluejeansAppSecret` with your credentials as their values. You can create them by running `/nc secret_create`.`

Remove grave accent (`) at the end as well.

Comment (niks3089) [reply bluejeans/README.md:16]: Done.

File: bluejeans/README.md [hunk 6]
```diff
@@ -12,14 +12,15 @@ Only supported on Mattermost for now.
...
 
 ## Usage
 
-To create a meeting with `satya@nimbella.com` & `eric@nimbella.com`:
+To create a meeting with `joe@example.com` & `judy@example.com`:
 ```sh
-/dapp bluejeans_create -title "A new beginning" -emails "satya@nimbella.com,eric@nimbella.com" -start "03/01/20 18:00" -end "03/01/20 18:30'
+/dapp bluejeans_create -title "A new beginning" -emails "joe@example.com,judy@example.com" -start "03/01/20 18:00" -end "03/01/20 18:30'
 ```
+Where date is in `mm/dd/yy` format and time is in UTC
```
Comment (satyarohith) [inline bluejeans/README.md]:
```suggestion
Where date is in `mm/dd/yy` format and time is in UTC.
```

File: bluejeans/src/bluejeans.js [hunk 7]
```diff
@@ -24,10 +24,11 @@ After you've the credentials, we need two secrets named \`bluejeansAppKey\` & \`
 
 ## Usage
 
-To create a meeting with \`satya@nimbella.com\` & \`eric@nimbella.com\`:
+To create a meeting with \`joe@example.com\` & \`judy@example.com\`:
 \`\`\`sh
-/dapp bluejeans_create -title "A new beginning" -emails "satya@nimbella.com,eric@nimbella.com" -start "03/01/20 18:00" -end "03/01/20 18:30'
+/dapp bluejeans_create -title "A new beginning" -emails "joe@example.com,judy@example.com" -start "03/01/20 18:00" -end "03/01/20 18:30'
 \`\`\`
+Where date is in `mm/dd/yy` format and time is in UTC
```
Comment (satyarohith) [inline bluejeans/src/bluejeans.js]: Update the changes in `bluejeans.js`.

**Review by satyarohith — APPROVED** [review]
LGTM.

**Pull request merged**